queries.yaml

```diff
@@ -1865,6 +1892,7 @@ post_process:
         - addr_housenumber
         - addr_street
         - osm_relation
+      exclude: ['superseded']
```

Comment:
if a building gets marked by a landmark, we exclude it from merging with other buildings, keeping it a single entity